pkg/deps/config/client.go

```diff
@@ -0,0 +1,81 @@
+package config
+
+import (
+	"fmt"
+
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/types"
+	"github.com/spf13/cobra"
+
+	"github.com/pokt-network/poktroll/app/volatile"
+	"github.com/pokt-network/poktroll/pkg/client"
+	"github.com/pokt-network/poktroll/pkg/client/tx"
+)
```

Comment:
I know @red-0ne had some recent related changes.

Wanted to make sure we're not duplicating logic?

Reply:
This derived from what @red-0ne wrote, it's mostly an extraction in the name of reusability (which is needed, not just nice to have, in this case).